app/login/register-template.html

```diff
@@ -24,7 +24,7 @@ <h2>Login Form</h2>
         <label><b>Password</b></label>
         <input type="password" placeholder="Enter Password" ng-model="pass" required>
         <div>
-            <button type="submit" ng-click="login(mail, pass)">Login</button>
+            <button type="submit" class="loginbtn" ng-click="login(mail, pass)">Login</button>
```

Comment:
use kebab case naming for classes, e.g `login-button`

Reply:
done